lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -148,9 +147,9 @@ def render_to_fragment(self, request, program_uuid, **kwargs):  # lint-amnesty,
             'certificate_data': certificate_data,
             'industry_pathways': industry_pathways,
             'credit_pathways': credit_pathways,
-            'program_discussions_enabled': program_discussion_lti.is_enabled,
+            'program_discussions_enabled': program_tab_view_is_enabled(),
             'discussion_fragment': {
-                'enabled': program_discussion_lti.is_configured,
+                'enabled': bool(program_discussion_lti.configuration),
```

Comment:
`enabled` => `is_configured`

Reply:
I guess this refactor should be deferred and moved to another ticket because to make things consistent in the frontend part we have to update a lot of code
 ie https://github.com/edx/edx-platform/search?q=programDiscussionEnabled&type=

Reply:
sure